src/test/java/com/github/alexcojocaru/mojo/elasticsearch/StartElasticsearchDataMojoTest.java

```diff
@@ -32,8 +32,8 @@ protected void tearDown() throws Exception
     public void testMojoLookup() throws Exception
     {
         File testPom = new File(getBasedir(), "src/test/resources/goals/start/pom.xml");
-        
-        StartElasticsearchNodeMojo mojo = (StartElasticsearchNodeMojo)lookupMojo("start", testPom);
```

Comment:
I disagree with the rename. The ES project is called Elasticsearch and not ElasticSearch.